README.md

```diff
@@ -22,7 +23,7 @@ A Puppet module to download files with curl, supporting authentication.
 
 # License
 
-Copyright (c) 2013 Government Digital Services
+Copyright (c) 2015 Government Digital Service
```

Comment:
The copyright year should be the year that the software was first release, so it doesn't need updating.

:+1: to the typo fix though.